CMakeLists.txt

```diff
@@ -80,6 +80,8 @@ set(IRODS_PLUGIN_TARGET_NAME "${IRODS_TARGET_NAME_PREFIX}-${IRODS_POLICY_NAME}")
 string(REPLACE "_" "-" IRODS_POLICY_NAME_HYPHENS "${IRODS_POLICY_NAME}")
 set(IRODS_POLICY_PACKAGE_COMPONENT "${IRODS_POLICY_NAME_HYPHENS}")
 
+add_subdirectory(test/stream_test)
```

Comment:
Perhaps this should require a cmake option to include this step. That way we don't include it in the release build.

I'm mostly thinking about how the binary doesn't have a limit on the size of the read buffer as well as not exposing a tool which a malicious user could gain access to.

Reply:
Good point. I'll look into adding an option a la `IRODS_UNIT_TESTS_BUILD`.

Reply:
Noting here that we should probably add the build option to the documentation somewhere, as well...

Reply:
Okay, took a shot at an implementation... Let's discuss that in a separate conversation.